tests/suites/test_suite_gcm.function

```diff
@@ -233,6 +233,74 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void gcm_encrypt_and_tag_reuse_ctx(int cipher_id, data_t *key_str,
```

Comment:
This only tests context reuse for encryption. I think while we're at it we should also test decryption.